omod/src/main/webapp/resources/scripts/conditionlist/controllers/editcondition.controller.js

```diff
@@ -0,0 +1,147 @@
+var app = angular.module("conditionApp",
+    ['uicommons.widget.coded-or-free-text-answer', 'app.restfulServices', 'app.models', 'app.commonFunctionsFactory']);
+
+app.controller("ConditionController", ConditionController);
+ConditionController.$inject = ['$scope', 'RestfulService', 'ConditionModel', 'ConceptModel', 'CommonFunctions'];
+
+function ConditionController($scope, RestfulService, ConditionModel, ConceptModel, CommonFunctions) {
+    var self = this;
+
+    $scope.patientUuid = null;
+    $scope.condition = null;
+    const INACTIVE_STATUS = 'Inactive';
+
+    // this is required inorder to initialize the Restangular service
+    RestfulService.setBaseUrl('/' + OPENMRS_CONTEXT_PATH + '/ws/rest/emrapi');
+
+    /**
+     * Perform POST save
+     * PS: Make sure the 'Non Coded Uuid' setting (under Settings->Condition List) is set inorder to work with noncoded concepts,
+     * the LOCALE needs to be set (most concepts have 'en')
+     * @type {Function}
+     */
+    self.saveCondition = self.saveCondition || function () {
+        if($scope.condition.onSetDate>$scope.condition.endDate){
```

Comment:
Can you put space around the > sign?

Reply:
Could you explain why this one does not have the inactive condition check like the one you have in the other pull request?

Reply:
Thank you for pointing this out. I updated the PR